restli-client/src/test/java/com/linkedin/restli/client/TestParSeqBasedCompletionStage.java

```diff
@@ -1302,6 +1325,75 @@ public void testWhenCompleteAsync() throws Exception
     assertTrue(waitLatch.await(1000, TimeUnit.MILLISECONDS));
   }
 
+  private void testWithcomposableApi(CompletionStage stage, List<CountDownLatch> latches) throws Exception {
+    Consumer<String> consumer = mock(Consumer.class);
+    CompletionStage<String> completionStage = createTestStage(TESTVALUE1);
+    if (latches == null)
+    {
+      // synchronization not needed, then wait blockingly
+      finish(completionStage.thenCompose(r -> stage).thenAccept(consumer));
+      verify(consumer, times(1)).accept(TESTVALUE2);
+    }
+    else
+    {
+      new Thread(() ->{
+        try {
```

Comment:
although it is tiny, but the code format seems a little bit inconsistent.
for line 1338, it is a bracket in new line, but in line 1339 1340 it is bracket in same line